linchpin/provision/action_plugins/ec2_vpc_endpoint.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.plugins.action import ActionBase
+import linchpin.MockUtils.MockUtils as mock_utils
+
+
+class ActionModule(ActionBase):
+    def run(self, tmp=None, task_vars=None):
+        """
+        Simple action plugin that returns the mocked output
+        when linchpin_mock is True
+        """
+        super(ActionModule, self).run(tmp, task_vars)
+        # contains all the module arguments
+        module_args = self._task.args.copy()
+        # task vars.keys() contains all the variable  required
+        # when passed a extra_var as key value pair task_vars
+        # would return mocked output of the named module.
+        # print(task_vars['vars'].keys())
```

Comment:
Can you delete these?  They're in a couple of the other files too

Reply:
@14rcole Sure I will update the PR